src/app/functions/server/getStuckRemovals.ts

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getAllScanResults } from "../../../db/tables/onerep_scans";
+import { getAllDataBrokers } from "./onerep";
+import { refreshStoredScanResults } from "./refreshStoredScanResults";
+
+export async function getStuckRemovals(
+  days: number,
+  page: number | null,
+  perPage: number | null,
+) {
+  const age = new Date();
+  const dateOffset = 24 * 60 * 60 * 1000 * days;
```

Comment:
We have the const `CONST_DAY_MILLISECONDS`. :)

```suggestion
  const dateOffset = CONST_DAY_MILLISECONDS * days;
```